### DISCUSSION TITLE
gerbera rereads all file with mtime == epoch

### DISCUSSION THREAD

**Discussion by nobanzai** [gerbera/gerbera#3266] [General]
Hi *,

files with mtime 1970-01-01 (epoch) are reread by gerbera on every restart.
Why?
Touching the files stops this cycle.

TIA.
Bye.
Michael.

**Comment by KarlStraussberger** [comment]
When restarting gerbera we need to detect whether the file was changed since last run.
1970-01-01 is simply considered as "missing date", so we rescan again.

**Comment by nobanzai** [comment]
Ok, thx.
